docs/content/preview/contribute/docs/all-page-elements.md

```diff
@@ -0,0 +1,414 @@
+
+    ```sh
+    $ java -cp target/yugabyte-simple-java-app-1.0-SNAPSHOT.jar SampleApp
+    ```
+
+The end of this tab.
+
+  {{% /tab %}}
+
+  {{% tab header="Go" lang="go" %}}
+
+The contents of the next tab. You can keep adding tabs in similar fashion.
+
+### Headings inside this widget don't show up in RightNav
+
+To build and run the application, do the following:
+
+1. Clone the sample application to your computer:
+
+    ```sh
+    git clone https://github.com/YugabyteDB-Samples/yugabyte-simple-go-app.git && cd yugabyte-simple-go-app
+    ```
+
+1. Provide connection parameters.
+
+  {{% /tab %}}
+
+{{< /tabpane >}}
+
+### Pills
+
+These pills automatically re-flow based on page width; just keep adding list items. Use these pills for landing pages or as an alternative to tabs.
+
+<ul class="nav yb-pills">
+
+  <li>
+    <a href="../../../develop/build-apps/java/cloud-ysql-yb-jdbc/" class="orange">
+      <i class="fa-brands fa-java"></i>
+      Java
+    </a>
+  </li>
+
+  <li>
+    <a href="../../../develop/build-apps/go/cloud-ysql-go/" class="orange">
+      <i class="fa-brands fa-golang"></i>
+      Go
+    </a>
+  </li>
+
+  <li>
+    <a href="../../../develop/build-apps/python/cloud-ysql-python/" class="orange">
+      <i class="fa-brands fa-python"></i>
+      Python
+    </a>
+  </li>
+
+  <li>
+    <a href="../../../develop/build-apps/nodejs/cloud-ysql-node/" class="orange">
+      <i class="fa-brands fa-node-js"></i>
+      NodeJS
+    </a>
+  </li>
+
+</ul>
+
+## Table
+
+The following is a basic markdown table.
+
+| Table | A column |
+| :--- | :--- |
+| A row | Another column in a table. Maybe to describe stuff. Might have bulleted lists etc, but that all has to be done using HTML. |
+| Another row | Another row in a table. Maybe to describe stuff. Might have bulleted lists etc, but that all has to be done using HTML. |
+| Another row | Another column in a table. Maybe to describe stuff. Might have bulleted lists etc, but that all has to be done using HTML. |
+
+## Notes and blockquote
+
+An ordinary paragraph.
+
+>**Blockquote**
+>
+>Blockquote text
+>
+>Some more blockquote text.
+
+{{< note title="Note" >}}
+
+This is an actual Note. Pay special attention to this thing, maybe.
+
+{{< /note >}}
+
+{{< tip title="Tip" >}}
+
+This is an actual Tip. Maybe there's a cool alternative to what you are doing.
+
+{{< /tip >}}
+
+{{< warning title="Warning" >}}
+
+This is an actual Warning. Whatever you are doing may threaten your data if you do it wrong.
+
+{{< /warning >}}
+
+## Images and video
+
+What follows is an image:
+
+![Connect using cloud shell](/images/yb-cloud/cloud-connect-shell.gif)
+
+What follows is an embedded YouTube video:
+
+{{< youtube id="qYMcNzWotkI" title="Deploy a fault tolerant cluster in YugabyteDB Managed" >}}
+
+## Lists
+
+What follows are some numbered steps, some code, text indented under a step:
+
+1. Numbered list with indented lists.
+
+    1. Second level numbered list indent.
```

Comment:
Two-space indents are easier, instead of four...

Reply:
Provided it renders correctly. In some situations I find 4 is needed for the indentation to work correctly...

Reply:
Four it is! And I just broke a list like this today, so your point is well taken. ;-)